### DISCUSSION TITLE
Dose all peaks in the eRegulon contain the motif of the TF?

### DISCUSSION THREAD

**Discussion by jase11** [aertslab/scenicplus#96] [Q&A]
In the eReuglon meta data of SCENICplus, we could know that how a transcription factor (TF) regulated a gene via some peaks. I know there is a motif enrichment test using pycisTarget on the SCENICplus workflow. But does that means all peaks in the eReuglon contain the motif of the TF? And does all the regulations between TF, peak and gene were supported by the exact presence of the motif for the peaks?

**Comment by SeppeDeWinter** [comment]
Hi @jase11 


I'll try to answer your questions:

```
But does that means all peaks in the eReuglon contain the motif of the TF?
```

After doing the motif enrichment analysis we take the top scoring regions for each motif. In other words, yes, we do assume that all these regions contain the motif of the TF.

Could you clarify this question a bit?

```
And does all the regulations between TF, peak and gene were supported by the exact presence of the motif for the peaks?
```

Best,

Seppe

**Reply by jase11** [reply]
Thank you so much for your reply!

Actually I was referring to the eRegulon metadata which looks like this：
![image](https://user-images.githubusercontent.com/27488572/216938061-30c7e6df-7114-4ac5-ab7a-88eab74f7e58.png)

It shows how the transcription factor regulate the genes via a group of peaks. And I would like to know if there is the TF’s motif in each of these peaks?

**Reply by SeppeDeWinter** [reply]
Hi @jase11 

Yes, indeed these are the peaks enriched for the motif of the TF.

Best,

Seppe

**Reply by jase11** [reply]
Thanks. May I add another question?

I have run SCENIC+ using the newest SCENIC+ motif collection, and I found that a TF can have more than one motif in the database. Even if all peaks contain motifs of the TF, is it possible that they contain different motifs? Is there a way to know exactly what motif each peak contains?
